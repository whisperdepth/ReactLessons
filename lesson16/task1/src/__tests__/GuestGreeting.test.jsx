import React from "react";
import { shallow } from "enzyme";
import GuestGreeting from "../GuestGreeting.jsx";

describe("GuestGreeting", () => {
  it("should display guest greeting", () => {
    const wrappedComponent = shallow(<GuestGreeting />);

    expect(wrappedComponent).toMatchSnapshot();
  });
});