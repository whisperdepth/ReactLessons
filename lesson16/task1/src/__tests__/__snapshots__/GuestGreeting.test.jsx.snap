// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GuestGreeting should display guest greeting 1`] = `
<h1>
  Hi stranger âœŒ Please login ðŸš€
</h1>
`;
